README.md

```diff
@@ -1,2 +1,401 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Module Overview
```

Comment:
I would change this to "Repository Overview". Then you can talk about modules later, but first and foremost we explain the user what this Git repo is.